(GH-8777) Reword here-string section for clarity (#8780)
MicrosoftDocs/PowerShell-Docs: reference/5.1/Microsoft.PowerShell.Core/About/about_Quoting_Rules.md

The quotation rules for here-strings are slightly different.

> A here-string is a single-quoted or double-quoted string surrounded by at signs
> (`@`) in which quotation marks are interpreted literally. A here-string spans
> multiple lines and always ends with a newline before the closing mark (`@'` for
> single-quoted or `@"` for double-quoted). The closing mark of a here-string must
> always be the first text on a line. Every line between the opening and closing
> marks of a here-string is interpreted as part of a single string even if they
> aren't enclosed in quotation marks.

Reviewer: ```suggestion
A here-string is a single-quoted or double-quoted string surrounded by at signs
(`@`). The opening and closing marks must be on their own line. Quotation marks
within a here-string are interpreted literally. A here-string spans multiple
lines and always ends with a newline before the closing mark. The closing mark of
a here-string must always be the first text on a line. Every line between the
opening and closing marks of a here-string is part of a single string.
```

'@
```

> A here-string contains all the text between the opening and closing marks. In
> the here-string, all quotation marks are interpreted literally. For example:

Reviewer: ```suggestion
> [!NOTE]
> The final newline character is part of the closing mark. It is not added to
> the here-string.

A here-string contains all the text between the opening and closing marks. In
the here-string, all quotation marks are interpreted literally. For example:
```
